csharp-advanced-topics/MemoryInCSharp/App/MemoryVsArrayBenchmarks.cs

```diff
@@ -0,0 +1,55 @@
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Order;
+using BenchmarkDotNet.Reports;
+using BenchmarkDotNet.Running;
+
+namespace App;
+
+[MemoryDiagnoser]
+[RankColumn]
+[Orderer(SummaryOrderPolicy.FastestToSlowest)]
+public class MemoryVsArrayBenchmarks
+{
+    private const int Size = 10000000; // 10 million
+    private readonly int[] _array = new int[Size];
+    private readonly Memory<int> _memory;
+
+   public MemoryVsArrayBenchmarks()
+    {
+        for (var i = 0; i < Size; i++)
+        {
+            _array[i] = i;
+        }
+        _memory = new Memory<int>(_array);
+    }
```

Comment:
We can one line the array creation using `Enumerable.Range(0, Size).ToArray()` the performance difference will be minimal and since this benchmark setup code it doesn't matter. This has the advantage of shrinking the code snippet in the article as well

Reply:
@jshergal Thanks, Done